source/packages/services/greengrass2-provisioning/package.json

```diff
@@ -36,20 +36,20 @@
     }
   },
   "devDependencies": {
-    "@aws-sdk/abort-controller": "3.38.0",
-    "@aws-sdk/client-dynamodb": "3.38.0",
-    "@aws-sdk/client-eventbridge": "3.38.0",
-    "@aws-sdk/client-greengrassv2": "3.38.0",
-    "@aws-sdk/client-iot": "3.38.0",
-    "@aws-sdk/client-lambda": "3.38.0",
-    "@aws-sdk/client-s3": "3.38.0",
-    "@aws-sdk/client-sqs": "3.38.0",
-    "@aws-sdk/lib-dynamodb": "3.38.0",
-    "@aws-sdk/lib-storage": "3.38.0",
-    "@aws-sdk/signature-v4-crt": "3.38.0",
-    "@aws-sdk/smithy-client": "3.38.0",
-    "@aws-sdk/types": "3.38.0",
-    "@aws-sdk/util-dynamodb": "3.38.0",
+    "@aws-sdk/abort-controller": "3.374.0",
```

Comment:
Why is this dependency downgraded?

Reply:
Looks like this specific package's latest version is in fact `3.374.0`. See their [npm package page](https://www.npmjs.com/package/@aws-sdk/abort-controller?activeTab=versions). And this package is marked as "deprecated" with the message `This package has moved to @smithy/abort-controller`.